geode-redis/src/main/java/org/apache/geode/redis/internal/ExecutionHandlerContext.java

```diff
@@ -56,7 +56,7 @@
   private static final Logger logger = LogService.getLogger();
   private static final int WAIT_REGION_DSTRYD_MILLIS = 100;
   private static final int MAXIMUM_NUM_RETRIES = (1000 * 60) / WAIT_REGION_DSTRYD_MILLIS; // 60
-                                                                                          // seconds
+  // seconds
```

Comment:
This one still does not look correct. The comment is not even correct (this is a max retry count not a time. The 1000*60 represents 60 seconds (in millis). You could extract that into another final int and name it 60_SECONDS_IN_MILLIS. The JDK also has some classes that can help with these time unit conversions

Reply:
I'll have to look at why that changed- I definitely didn't change any comments intentionally